doc/usage/tutorial.md

```diff
@@ -2,53 +2,99 @@
 
 `kube-batch` need to run as a kubernetes scheduler. The next step will show how to run `kube-batch` as kubernetes scheduler quickly. Refer [Configure Multiple Schedulers](https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers/) to get more details.
 
-## 2. Config kube-batch for Kubernetes
+## Install kube-batch for Kubernetes
 
-### (1) kube-batch image
-
-An official kube-batch image is provided and you can download it from [DockerHub](https://hub.docker.com/r/kubesigs/kube-batch/). The version is `v0.4.2` now.
-
-```bash
-# docker pull kubesigs/kube-batch:v0.4.2
-```
-
-### (2) Create a Kubernetes Deployment for kube-batch
-
-#### Download kube-batch
+### Download kube-batch
 
 ```bash
 # mkdir -p $GOPATH/src/github.com/kubernetes-sigs/
 # cd $GOPATH/src/github.com/kubernetes-sigs/
 # git clone http://github.com/kubernetes-sigs/kube-batch
 ```
 
-#### Deploy `kube-batch` by Helm
+### Deploy kube-batch by Helm
 
-Run the `kube-batch` as kubernetes scheduler
+Run `kube-batch` as kubernetes scheduler. The latest kube-batch version is `v0.4.2`.
 
 ```bash
 # helm install $GOPATH/src/github.com/kubernetes-sigs/kube-batch/deployment/kube-batch --namespace kube-system
 ```
 
-Note: If there is an error `Error: apiVersion "scheduling.incubator.k8s.io/v1alpha1" in kube-batch/templates/default.yaml is not available`, please update your helm to latest version and try it again.
-
 Verify the release
 
 ```bash
 # helm list
 NAME        	REVISION	UPDATED                 	STATUS  	CHART                	NAMESPACE
-dozing-otter	1       	Thu Jun 14 18:52:15 2018	DEPLOYED	kube-batch-0.4.1    	kube-system
+dozing-otter	1       	Thu Mar 29 18:52:15 2019	DEPLOYED	kube-batch-0.4.2   	 kube-system
 ```
 
-NOTE: `kube-batch` need to collect cluster information(such as Pod, Node, CRD, etc) for scheduling, so the service account used by the deployment must have permission to access those cluster resources, otherwise, `kube-batch` will fail to startup. For users who are not familiar with Kubernetes RBAC, please copy the example/role.yaml into `$GOPATH/src/github.com/kubernetes-sigs/kube-batch/deployment/kube-batch/templates/` and reinstall batch.
+## Customize kube-batch
+
+Please check following configurable fields and customize kube-batch based on your needs.
+
+### kube-batch configuraions
+| configs | default value | Descriptions |
+| ----------- | ----------- | ------ | ----------- |
+| master | "" | The address of the Kubernetes API server (overrides any value in kubeconfig) |
+| kubeconfig | "" | Path to kubeconfig file with authorization and master location information. If you run kube-batch outside cluster or for debug purpose, this is very useful |
+| scheduler-name | kube-batch | kube-batch will handle pods with the scheduler-name |
```

Comment:
kube-batch will handle pods whose `.spec.SchedulerName` is  same as scheduler-name